src/foldermodel.cpp

```diff
@@ -224,7 +224,7 @@ QString FolderModel::makeTooltip(FolderModelItem* item) const {
         tip += QStringLiteral("<p><i>") + tr("Location:") + QStringLiteral("</i> ") + QString::fromUtf8(parent_str.get()) + QStringLiteral("</p>");
     }
     // file type
-    tip += QStringLiteral("<i>") + tr("File type:") + QStringLiteral("</i> ") + info->mimeType()->desc();
+    tip += QStringLiteral("<i>") + tr("File type:") + QStringLiteral("</i> ") + QLatin1String(info->mimeType()->desc());
```

Comment:
Wouldn't `QString::fromUtf8(info->mimeType()->desc())` be more consistent with other changes?

Reply:
Yes, it should. That one escaped.